rules/typescript.js

```diff
@@ -11,6 +11,7 @@ const { rules: baseVariablesRules } = require('./variables');
  * - no-unsafe-member-access
  */
 module.exports = {
+  plugins: ['@typescript-eslint'],
```

Comment:
This setting isn't required since it's included in `@typescript-eslint/strict-type-checked` (or `@typescript-eslint/stylistic-type-checked`),  but is specified for consistency with other rulesets.